plugins/organic/organic.cpp

```diff
@@ -325,11 +326,7 @@ void organicInstrument::playNote( NotePlayHandle * _n,
 
 void organicInstrument::deleteNotePluginData( NotePlayHandle * _n )
 {
-	delete static_cast<Oscillator *>( static_cast<oscPtr *>(
-						_n->m_pluginData )->oscLeft );
-	delete static_cast<Oscillator *>( static_cast<oscPtr *>(
-						_n->m_pluginData )->oscRight );
-	
+	MM_FREE( static_cast<oscPtr *>(_n->m_pluginData )->oscLeft);
```

Comment:
In analogy to the allocation, this should be written as
```cpp
delete[] static_cast<oscPtr *>(_n->m_pluginData )->oscLeft
```

Using `MM_FREE` won't call the object's destructors.